jekyll/_layouts/classic-docs.html

```diff
@@ -7,14 +7,28 @@
 {% endif %}
 
 <div class="outer">
-  {% include_cached global-nav.html site=site page=page %}
+  {% assign lang = page.lang %}
+  {% assign sidenavcci2 = site.data.sidenav[lang] %}
+  {% assign sidenavstyle = site.data.sidenavstyle[lang] %}
+
+  {% if page.collection == "cci2" %}
+    {% include_cached global-nav.html site=site page=page sidenav=sidenavcci2 current_url=current_url %}
+  {% elsif page.collection == "style" %}
+    {% include_cached global-nav.html site=site page=page sidenav=sidenavstyle current_url=current_url %}
+  {% else %}
+    {% include_cached global-nav.html site=site page=page sidenav=sidenavcci2 current_url=current_url %}
+  {% endif %}
+
   {% include_cached hits-target.html %}
   <div class="main-body">
-    {% assign lang = page.lang %}
-    {% assign sidenav = site.data.sidenav[lang] %}
-    {% assign current_url = page.url | slice: 1, page.url.size %}
-    {% include_cached sidebar.html lang=lang sidenav=sidenav current_url=current_url%}
-
+      {% assign current_url = page.url | slice: 1, page.url.size %}
+      {% if page.collection == "cci2" %}
+        {% include_cached sidebar.html lang=lang sidenav=sidenavcci2 current_url=current_url %}
+      {% elsif page.collection == "style" %}
+        {% include_cached sidebar.html lang=lang sidenav=sidenavstyle current_url=current_url %}
+      {% else %}
+        {% include_cached sidebar.html lang=lang sidenav=sidenavcci2 current_url=current_url %}
+      {% endif %}
```

Comment:
Can we simplify all of that? Can we not just conditionally assign the right data to `sidenav`? Something like:

```
{% if page.collection == "style" %}
    {% assign sidenav = site.data.sidenavstyle[lang] %}
{% else %}
    {% assign sidenav = site.data.sidenav[lang] %}
{% endif %}
```

and continue with the rest as usual?

Reply:
Oh That is a good point!